test/db/formats/elf/m68k

```diff
@@ -1,6 +1,14 @@
 NAME=ELF: m68k.elf
+BROKEN=1
 FILE=bins/elf/analysis/busybox.m68k
 CMDS=<<EOF
+# This test is broken. The reason is that Rizin can't distinguish
+# CPU models. Each M68k model has a different address mask (32-24 bits).
+# It defaults to one which has only 24bits.
+# Hence, the subi.l instruction decodes to
+# `subi.l 0x15b119, d0` instead of `subi.l 0x8015b119, d0`
+# Should be fixable with RzArch (if anyone is still intereses).
```

Comment:
Open an issue instead and link to it here.

Reply:
let me handle this, i think it can be resolved in the same way as for mips, because it looks like the exact same issue i faced.